manifests/integreatly-3scale/3scale-0.4.0/apps_v1alpha1_apimanager_crd.yaml

```diff
@@ -37,19 +37,49 @@ spec:
                   type: string
                 openSSLVerify:
                   type: boolean
+                productionSpec:
```

Comment:
I wonder if any of these new CR fields would be useful for us to set? :thinking:

Reply:
I'm not too sure on this one to be honest, don't know much on 3scale :see_no_evil: